traitsui/examples/demo/Standard_Editors/tests/test_FileEditor_demo.py

```diff
@@ -0,0 +1,61 @@
+# (C) Copyright 2004-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+"""
+This example demonstrates how to test interacting with a textbox created
+using FileEditor.
+
+The GUI being tested is written in the demo under the same name (minus the
+preceding 'test') in the outer directory.
+"""
+
+import os
+import runpy
+import unittest
+
+from traitsui.testing.api import (
+    DisplayedText, KeyClick, KeySequence, UITester
+)
+
+#: Filename of the demo script
+FILENAME = "FileEditor_demo.py"
+
+#: Path of the demo script
+DEMO_PATH = os.path.join(os.path.dirname(__file__), "..", FILENAME)
+
+
+class TestFileEditorDemo(unittest.TestCase):
+
+    def test_run_demo(self):
```

Comment:
You say in the PR description: "The test for the FileEditor demo fails on wx due to #752".  Should this have an `@requires_toolkit(['ToolkitName.qt'])` in that case?
/ a comment regarding the failure being linked to having a custom editor included in the demo?

Reply:
Good point. Yes I should add the decorator.

Reply:
> Yes I should add the decorator.

On second thought, this is a demo test so I'd want to avoid importing anything from `traitsui.tests._tools`.
Will go with a comment instead :/